Added due date to the index page

bartenbach opened the pull request:
Index goals page now shows the due date of the goal

bartenbach committed:
Added due date to the index page

bartenbach committed:
Fixed application name :)

silvercorked commented on src/main/resources/templates/goals/index.html:
```diff
@@ -58,9 +61,12 @@
                 <div class="col-4">
```
I think right here we got 3 col 4s and a col 2, giving 14. So might make the page look kinda funky. Prob could make the goal 3, dueDate 2 and description 5, since the description is pretty big and dueDate is always the same size

bartenbach replied:
Fixed by Cole's commit

silvercorked reviewed (commented):
I think we should prob talk about this part before adding to master. small thing

malycole committed:
Fixed Formatting Issues

Modified both Tags and Goals index to address formatting imperfections. Should look a little better.

malycole reviewed (dismissed):
Looks good

silvercorked commented on src/main/resources/templates/goals/index.html:
```diff
@@ -28,16 +28,19 @@
-                <hr width="60%" align="left">
+                <hr width="100%" align="left">
             </div>
             <div class="row">
```
Columns here still don't sum to 12

silvercorked commented on src/main/resources/templates/goals/index.html:
```diff
@@ -55,16 +58,19 @@
-                <div class="col-4">
+                <div class="col-3">
                     <p th:text="${goal.goal}"></p>
                 </div>
```
cols should sum to 12

silvercorked commented on src/main/resources/templates/tags/index.html:
```diff
@@ -67,7 +67,7 @@
+                <hr width="100%" align="left">
             </div>
             <div class="row">
                 <div class="col-4">
```
cols here don't sum to 12. Could also use a little css instead of a <hr>, but that's just my opinion

malycole replied:
What would you recommend for the CSS, is there something you have in mind or just anything that is better than <'hr'>?

silvercorked replied:
I usually go with a class that gives a border bottom or something like that. 
`.mySeperator {
  border-bottom: 1px solid #{hexColorCode or name of common color}
}`
It's more customizable, but if <hr> works then it aint a big deal

malycole replied:
So it works as is right now, but I think it would be valuable to use the most up to date or effective methods. Do we have a stylesheet currently that I'm missing or should we create one?

silvercorked replied:
I actually am not sure of the best methods for linking a stylesheet properly with spring. We had one before that had trouble linking the conventional html way earlier. Feel free to look into it and make a stylesheet if you'd like. We don't currently have one. You could potentially make a fragment (see the header file and how it is included in pages) and simply put css in there as well as the links for bootstrap and jquery. Might be a good idea

malycole replied:
I will go ahead and make a separate branch for that so that we can get this pull in. It seems to have been waiting for a bit and fixes a few things.

silvercorked reviewed (commented):
Still some column issues on this one

malycole committed:
Modified col size

All cols should equal 12

silvercorked reviewed (approved):
cool, looks good
